muxers/mplex/src/codec.rs

```diff
@@ -18,14 +18,10 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use asynchronous_codec::{Decoder, Encoder};
```

Comment:
Please revert the code formatting introduced in https://github.com/libp2p/rust-libp2p/pull/2094/commits/a01622283045025a33f6578082b5ed461c7e0372.

I am not opposed to using rustfmt, though I think, in case we do decide to use it across the repository, it should be a separate pull request. See https://github.com/libp2p/rust-libp2p/issues/1668 for details.

Reply:
I had accidentally used `rustfmt` in the commit [ead5d97](https://github.com/libp2p/rust-libp2p/pull/2094/commits/ead5d97ae7d18dbc86abe7f5087d8e5deba00608). The commit [a016222](https://github.com/libp2p/rust-libp2p/pull/2094/commits/a01622283045025a33f6578082b5ed461c7e0372) reverted the formatting that was done by `rustfmt`.